### DISCUSSION TITLE
403 errors after VM recovery

### DISCUSSION THREAD

**Discussion by sgallaga-woodkh** [datalust/seq-tickets#1794] [Help]
Hi!

We have been running Seq on a windows server with IIS that recently had a number of system failures requiring us to clone and recreate the VM using a disk snapshot applied onto a new VM with a different name. Since the recovery we have been unable to get Seq working again. We had Seq configured using virtual directories, with lower environments pointing to https://seq.<domainroot>/<env> and prod pointing to https://seq.<domainroot>. At prod we see the pre-load site (Seq logo in the top-left corner and the shifting gradient background animation) and get the following error in the console: 
`mvendor-1678318981997.min.js:7 Possibly unhandled rejection: {"data":"<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">\r\n<html xmlns=\"http://www.w3.org/1999/xhtml\">\r\n<head>\r\n<meta http-equiv=\"Content-Type\" content=\"text/html; charset=iso-8859-1\"/>\r\n<title>404 - File or directory not found.</title>\r\n<style type=\"text/css\">\r\n<!--\r\nbody{margin:0;font-size:.7em;font-family:Verdana, Arial, Helvetica, sans-serif;background:#EEEEEE;}\r\nfieldset{padding:0 15px 10px 15px;} \r\nh1{font-size:2.4em;margin:0;color:#FFF;}\r\nh2{font-size:1.7em;margin:0;color:#CC0000;} \r\nh3{font-size:1.2em;margin:10px 0 0 0;color:#000000;} \r\n#header{width:96%;margin:0 0 0 0;padding:6px 2% 6px 2%;font-family:\"trebuchet MS\", Verdana, sans-serif;color:#FFF;\r\nbackground-color:#555555;}\r\n#content{margin:0 0 0 2%;position:relative;}\r\n.content-container{background:#FFF;width:96%;margin-top:8px;padding:10px;position:relative;}\r\n-->\r\n</style>\r\n</head>\r\n<body>\r\n<div id=\"header\"><h1>Server Error</h1></div>\r\n<div id=\"content\">\r\n <div class=\"content-container\"><fieldset>\r\n  <h2>404 - File or directory not found.</h2>\r\n  <h3>The resource you are looking for might have been removed, had its name changed, or is temporarily unavailable.</h3>\r\n </fieldset></div>\r\n</div>\r\n</body>\r\n</html>\r\n","status":404,"config":{"method":"GET","transformRequest":[null],"transformResponse":[null],"jsonpCallbackParam":"callback","url":"api","headers":{"Accept":"application/json, text/plain, */*"}},"statusText":"Not Found","xhrStatus":"complete"}`

For the low env sites we get an IIS error page with the following error :
`403 - Forbidden: Access is denied.
You do not have permission to view this directory or page using the credentials that you supplied.`

The seq application logs (found in programdata\seq\logs) don't seem to be logging any errors.

I've tried re-installing seq, doing seq service uninstall/install for each of the named instances and changing the virtual directory to connect using a local admin (with confirmed access to the programdata\seq\instance\ dirs) but can't seem to make any progress here. Do you know of anything else I could try, or have any additional troubleshooting ideas for me to try to get to the root of the issue? Thanks!

-Shane

**Comment by nblumhardt** [comment]
Hi Shane, thanks for dropping us a line!

Nothing obvious jumps out; the best option might be to drop `support@datalust.co` a line so we can jump on a call and look at it with you. I'll keep an eye on the inbox now while we're in AEST business hours.

Best regards,
Nick

**Reply by sgallaga-woodkh** [reply]
Hi Nick, I just sent this over to that support email. Thanks for the follow-up!
